contracts/contracts/gho/gho-core/src/contracts/gho/GhoToken.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {EnumerableSet} from "openzeppelin-contracts-gho/utils/structs/EnumerableSet.sol";
+import {AccessControl} from "openzeppelin-contracts-gho/access/AccessControl.sol";
+import {ERC20} from "./ERC20.sol";
+import {IGhoToken} from "./interfaces/IGhoToken.sol";
+
+/**
+ * @title GHO Token
+ * @author Aave
+ */
+contract GhoToken is ERC20, AccessControl, IGhoToken {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    mapping(address => Facilitator) internal _facilitators;
+    EnumerableSet.AddressSet internal _facilitatorsList;
+
+    /// @inheritdoc IGhoToken
+    bytes32 public constant FACILITATOR_MANAGER_ROLE =
+        keccak256("FACILITATOR_MANAGER_ROLE");
+
+    /// @inheritdoc IGhoToken
+    bytes32 public constant BUCKET_MANAGER_ROLE =
+        keccak256("BUCKET_MANAGER_ROLE");
+
+    /**
+     * @dev Constructor
+     * @param admin This is the initial holder of the default admin role
+     */
+    constructor(address admin) ERC20("Gho Token", "GHO", 18) {
+        _setupRole(DEFAULT_ADMIN_ROLE, admin);
+    }
```

Comment:
assign facilitator manager role in constructor so that we can add facilitators and mint GHO

Reply:
This is the original source code of GHO Token, do we want to change that?
I am right now working on a script that will add the facilitator using the addFacilitator function, which I think would be the ideal way.